[CUBRIDMAN-234] Add broker parameters for ACL per broker (#545)
CUBRID/cubrid-manual: en/admin/control.rst

----------------------

To limit the client applications accessing the broker, set to **ON** for the **ACCESS_ CONTROL** parameter in the **cubrid_broker.conf** file, and enter a name of the file in which the users and the list of databases and IP addresses allowed to access the **ACCESS_CONTROL_FILE** parameter value are written. 

> The default value of the **ACCESS_CONTROL** broker parameter is **OFF**. If there is no broker name in ACCESS_CONTROL_FILE, all access to that broker will be restricted. In this case, you can allow all access by setting the ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER parameter.

Reviewer: The default value of the **ACCESS_CONTROL** broker parameter is **OFF**. All access to brokers not listed in ACCESS_CONTROL_FILE is restricted. Even not listed in ACCESS_CONTROL_FILE, you can allow access to a specific broker by setting ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER to ALLOW for that broker.

Author: 수정했습니다.

> The default value of the **ACCESS_CONTROL** broker parameter is **OFF**. All access to brokers not listed in **ACCESS_CONTROL_FILE** is restricted. Even not listed in **ACCESS_CONTROL_FILE**, you can allow access to a specific broker by setting **ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER** to **ALLOW** for that broker.

Reviewer 2: ```suggestion
The default value of the **ACCESS_CONTROL** broker parameter is **OFF**. All access to brokers not listed in **ACCESS_CONTROL_FILE** is restricted. Even if not listed in **ACCESS_CONTROL_FILE**, you can allow access to a specific broker by setting **ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER** to **ALLOW** for that broker.
```

Author: 수정했습니다.

> The **ACCESS_CONTROL** and **ACCESS_CONTROL_FILE** parameters must be written under [broker] which common parameters are specified and **ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER** parameter must be written for each broker.

Reviewer: The **ACCESS_CONTROL** and **ACCESS_CONTROL_FILE** parameters must be written under [broker] which common parameters are specified. On the other hand, **ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER** parameter must be specified for each broker.

Author: 수정했습니다.

> The **ACCESS_CONTROL** and **ACCESS_CONTROL_FILE** parameters must be written under [broker] which common parameters are specified. On the other hand, **ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER** parameter must be specified for each broker.

Reviewer 2: ```suggestion
The **ACCESS_CONTROL** and **ACCESS_CONTROL_FILE** parameters must be written under the [broker] section where common parameters are specified. On the other hand, the **ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER** parameter must be specified for each broker.
```

Author: 수정했습니다.

*   <ip_addr>: An IP address that is allowed to access the server. If the last digit of the address is specified as \*, all IP addresses in that rage are allowed to access the broker server.

If a value for **ACCESS_CONTROL** is set to ON and a value for **ACCESS_CONTROL_FILE** is not specified, the broker will only allow the access requests from the localhost. 

> However, even **ACCESS_CONTROL_FILE** is not specified, all requests are allowed for brokers with **ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER** set to **ALLOW**.

Reviewer 2: ```suggestion
However, if **ACCESS_CONTROL_FILE** is not specified, all requests are allowed for brokers with **ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER** set to **ALLOW**.
```

Author: 수정했습니다.

> However, if **ACCESS_CONTROL_FILE** is not specified, for brokers, if **ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER** is set to ALLOW, the broker will allow all access requests.

Reviewer: However, even **ACCESS_CONTROL_FILE** is not specified, all requests are allowed for brokers with **ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER** set to **ALLOW**.

Author: 수정했습니다.

Broker access restrictions not specified in **ACCESS_CONTROL_FILE**.

* Allow access only from localhost. (default)

> * If ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER is set to ALLOW, all access is allowed.

Reviewer 2: ```suggestion
* If **ACCESS_CONTROL_BEHAVIOR_FOR_EMPTYBROKER** is set to **ALLOW**, all access is allowed.
```

Author: 수정했습니다.
